perfdash/perfdash.go

```diff
@@ -57,9 +58,10 @@ func initGoogleDownloaderOptions() {
 }
 
 func main() {
-	fmt.Println("Starting perfdash...")
+	klog.InitFlags(nil)
+	klog.Infoln("Starting perfdash...")
 	if err := run(); err != nil {
-		fmt.Fprintln(os.Stderr, err)
+		klog.Errorln(err)
```

Comment:
Do we need the `Errorln`? From the docs https://godoc.org/github.com/kubernetes/klog#Errorf a newline is appended when missing.

Won't klog.Error be good enough here?

Reply:
Thank you. I have changed it.